libs/platform/unit/platform_unit_test.cpp

```diff
@@ -419,8 +419,8 @@ static NTSTATUS
 test_provider_attach_client(
     HANDLE nmr_binding_handle,
     _Inout_ void* provider_context,
-    _In_ PNPI_REGISTRATION_INSTANCE client_registration_instance,
-    _In_ void* client_binding_context,
+    _In_ const PNPI_REGISTRATION_INSTANCE client_registration_instance,
```

Comment:
I don't believe this is correct.  See https://stackoverflow.com/questions/2253738/c-typedef-interpretation-of-const-pointers and other discussions.   What you have is an immutable pointer to a mutable structure, rather than a pointer to an immutable structure.  I think you would need:
```suggestion
    _In_ const NPI_REGISTRATION_INSTANCE* client_registration_instance,
```
to get a pointer to a const struct.

Reply:
OK, I think I found and fixed all of them. If you see any I missed, please flag them.